src/backend/backend.c

```diff
@@ -406,6 +406,18 @@ void paint_all_new(session_t *ps, struct managed_win *t, bool ignore_damage) {
 			                               &reg_visible);
 			ps->backend_data->ops->release_image(ps->backend_data, new_img);
 			pixman_region32_fini(&reg_visible_local);
+
+			// Blur window
+			if (w->blur_foreground) {
+				assert(ps->o.blur_method != BLUR_METHOD_NONE);
+				// FIXME Think more about combining blur w/ opacity
+				// FIXME Don't hardcode opacity
+				//        - needs to be set when focus_mode set to true
```

Comment:
I'd like to see these resolved before merging this.

Reply:
Added an `--inactive-blur-opacity` option for this in 45dc995